pkg/admissioncontroller/apis/config/v1alpha1/defaults_test.go

```diff
@@ -24,69 +24,101 @@ import (
 )
 
 var _ = Describe("Defaults", func() {
-	Describe("AdmissionControllerConfiguration", func() {
-		var obj *AdmissionControllerConfiguration
+	Describe("AdmissionControllerConfiguration defaulting", func() {
+		It("should default AdmissionControllerConfiguration correctly", func() {
+			obj := &AdmissionControllerConfiguration{}
+			SetObjectDefaults_AdmissionControllerConfiguration(obj)
+			// ContentType fields will be defaulted by client constructors / controller-runtime based on whether a
+			// given APIGroup supports protobuf or not. defaults must not touch these, otherwise the integelligent
+			// logic will be overwritten
+			Expect(obj.GardenClientConnection.ContentType).To(BeEmpty())
+			Expect(obj.GardenClientConnection.AcceptContentTypes).To(BeEmpty())
+			Expect(obj.GardenClientConnection).To(Equal(componentbaseconfigv1alpha1.ClientConnectionConfiguration{
+				QPS:   50.0,
+				Burst: 100,
+			}))
+			Expect(obj.LogLevel).To(Equal("info"))
+			Expect(obj.LogFormat).To(Equal("json"))
 
-		Context("Empty configuration", func() {
-			BeforeEach(func() {
-				obj = &AdmissionControllerConfiguration{}
-			})
+		})
 
-			It("should correctly default the admission controller configuration", func() {
-				SetObjectDefaults_AdmissionControllerConfiguration(obj)
+		It("should not default other fields that are set", func() {
```

Comment:
```suggestion
		It("should not overwrite already set values for admission controller configuration", func() {
```

Similar changes everywhere. This is for consistency as we use similar statement in other tests as well.